### PR TITLE
Update with 2019 brand colors and logo

### PR DESCRIPTION
**Pull request by feliciazhang**
Just added the new colors and changed the logo.

I'm not really sure how to run it? so not sure if it worked/can't get screenshots

### CODE REVIEW

**Review by warren — APPROVED** [review]

**Review by corteggiano — APPROVED** [review]

File: css/main.scss [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
-  font-family: 'Lora', serif;
-  color: $navy;
+  background-color: $gray;
+  font-family: 'Nunito', sans-serif;
+  color: $navy19;
 }
 
 h1 {
-  font-family: 'Montserrat', sans-serif;
+  font-family: 'Dosis', sans-serif;
```
Comment (ctlnwng) [inline css/main.scss]: We only use Dosis for our lockup (i.e. "HackBeanpot" with logo). So all headings/text should be Nunito!

File: css/main.scss [hunk 2]
```diff
@@ -29,24 +29,25 @@ h2 {
 }
 
 h3 {
-  font-family: 'Montserrat', sans-serif;
+  font-family: 'Dosis', sans-serif;
```
Comment (ctlnwng) [inline css/main.scss]: Same as above.

**Review by ctlnwng — CHANGES_REQUESTED** [review]
Just have one comment!

File: css/main.scss (hunk 1 above)
Comment (feliciazhang) [reply css/main.scss]: thanks, fixed now!

**Review by ctlnwng — APPROVED** [review]

**Pull request merged**